pss/pss.go

```diff
@@ -27,19 +27,28 @@ import (
 // Pss is the top-level struct, which takes care of message sending
 type Pss struct {
 	localStore chunk.Store
+	tags       *chunk.Tags
+}
+
+// Monitor is used for tracking status changes in sent trojan chunks
+type Monitor struct {
+	state chan chunk.State
```

Comment:
why do you need the chunk in this struct? 
or the tag? any good reason?
Maybe just make the monitor just a channel?

Reply:
I return the Monitor struct with chunk and tag that were created within Send;
So we can check if the chunk was inserted into localstore accordingly (only in the test), also for accessing the tag that was created. Maybe this should not be accessed from outside the function.

Should this be removed from the struct and used addressed through reflection?

If the chunk/tag is not returned from Send, how can I verify chunk/tag from outside if it's not being exported?

Reply:
- that the chunk is inserted into local store should be done within the test anyway , now you are using a feature to test another feature. You can get the newly added chunk from the localstore with an iterator.
- returning the tag makes a bit more sense, but i would say, either return the tag and leave it entirely to the application to do monitoring or just monitor with the state channel . 
- the tag should also be found from Tags object with iteration. 
- either way, if you leave these fields in, you must also export them :)

Reply:
removed the tag and chunk from Monitor.
Made the state an exported field.

Used the iteration to get the chunk and also the tag in the test.